sdk/src/main/java/com/mapbox/maps/renderer/widget/WidgetPosition.kt

```diff
@@ -6,11 +6,15 @@ import com.mapbox.maps.MapboxExperimental
  * Specifies widget position relative to the screen.
  * @param horizontal position relative to the screen.
  * @param vertical position relative to screen.
+ * @param offsetX horizontal offset of the widget.
+ * @param offsetY vertical offset of the widget.
  */
 @MapboxExperimental
 class WidgetPosition(
```

Comment:
Remember to add `equals` and `hashCode` (maybe `toString`)

Reply:
added